webserver/templates/user/resettoken.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html' %}
+
+{% block title %}Reset token - ListenBrainz{% endblock %}
+
+{% block content %}
+  <h3 class="page-title">Reset token</h3>
+  <p>Are you sure you want to reset your token? If you do, you will need
+  to change it any apps that you have added it to.</p>
+  <form method="post">
```

Comment:
I think this verbiage is a bit scant. How about?

Are you sure you want to reset your token?

If you do, you will need to replace your old token with the newly generated token in any of your applications that use your token. The old token will stop working immediately. Be sure to update the token immediately in your applications to ensure that you have no disruptions in recording your listens.